sympy/polys/polytools.py

```diff
@@ -4551,6 +4577,36 @@ def degree_list(f, *gens, **args):
 
     return tuple(map(Integer, degrees))
 
+@public
+def power_list(f, *gens, **args):
+    """
+    Return a list of powers of ``f`` in all variables.
+
+    Examples
+    ========
+
+    >>> from sympy import power_list
+    >>> from sympy.abc import x, y
+
+    >>> power_list(x**2 + y*x + 1)
+    [(2, 1, 0), (1, 0)]
+    >>> power_list(0, x)
+    [(0,)]
+    >>> power_list(x, x)
+    [(1,)]
+
+    """
+    options.allowed_flags(args, ['polys'])
+
+    try:
+        F, opt = poly_from_expr(f, *gens, **args)
+    except PolificationFailed as exc:
+        raise ComputationFailed('power_list', 1, exc)
+
+    powers = F.power_list()
+
+    return list(powers)
```

Comment:
In keeping with `degree_list` method vs function, the function return values should be sympified, so `return sympify(list(powers))` will return Tuples of Integers.

@jksuom , what do you think about adding this function/method? I think the [ambiguous output has been resolved](https://github.com/sympy/sympy/pull/17043#issuecomment-504357250) and the function will find use at least in the Lambert work that is taking place. It may be of pedagogical interest, too.